packages/core-js-builder/index.d.ts

```diff
@@ -0,0 +1,29 @@
+type Summary = {
+    size: boolean,
+    modules: boolean
+};
+
+type Options = {
+    /** entry / module / namespace / an array of them, by default - all `core-js` modules */
+    modules?: string[],
```

Comment:
It can be a string, regex, or an array of them, see https://github.com/zloirock/core-js/tree/master/packages/core-js-compat

Reply:
Thank you, resolved :)